core-java-modules/core-java-11-2/pom.xml

```diff
@@ -12,6 +12,7 @@
         <groupId>com.baeldung</groupId>
         <artifactId>parent-modules</artifactId>
         <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
```

Comment:
Do we really need it?

Reply:
When I tried executing the build on my local, maven started complaining about the relativePath in these 2 projects.